src/styles/theme.ts

```diff
@@ -27,18 +28,23 @@ const colors = {
   },
   background: {
     primary: "#2C3333",
-    secondary : "#4f5757",
-    third:"#EDF2F7"
+    secondary: "#4f5757",
+    third: "#EDF2F7",
   },
   fontcolor: {
     primary: "#C8C5C5",
-    secondary:"#1A202C",
+    secondary: "#1A202C",
+    third: "#4285f4",
```

Comment:
Do you think there should be 'tertiary' instead of 'third'?

Reply:
Sure, followed "background.third", didn't think through it. Will change.